owasp-modsecurity/ModSecurity-nginx
# feat: added Github CI workflow

Added `.github/workflow/ci.yml` and necessary files with a complex test workflow for each PR.

A brief summary about how does it work:
* set up dependencies (required packages)
* install latest ModSecurity3 release
* get ModSecurity-nginx source
* get latest Nginx soure
* build Nginx with ModSecurity module
* start Nginx with ModSecurity
* there are two virtual hosts
* there is only one rule
* run a valid request (expected code is 200) and an attack (expected code is 403) against both vhosts
